assets/templates/partials/json-ld/name.tmpl

```diff
@@ -0,0 +1,5 @@
+{{ if hasField .Metadata "Title" -}}
```

Comment:
I thought we were going to map that we a one to one mapping (e.g. id, title) in the handler?

Reply:
As per our call yesterday I've now removed the `name` partial as it was superfluous; the name property now directly takes from `Metadata.Title`